src/utils/v2v3/fundingCycle.ts

```diff
@@ -95,3 +100,36 @@ export const isValidMustStartAtOrAfter = (
     .add(fundingCycleDuration)
     .lt(MaxUint54)
 }
+
+// Derives next FC's issuance rate from previous FC's weight and discount
+export const deriveNextIssuanceRate = ({
+  weight,
+  previousFC,
+}: {
+  weight: BigNumber
+  previousFC: V2V3FundingCycle | undefined
+}) => {
+  const previousWeight = previousFC?.weight
+  // When setting issue rate to 0, we pass 1 to contracts
+  if (weight.eq(BigNumber.from(1))) {
+    return BigNumber.from(0)
```

Comment:
**suggestion** to make this a bit more clear, maybe we should have constants for 0 and one.

```js
const WEIGHT_ZERO = 1 // send `1` when we want to set the weight to `0`
const WEIGHT_UNCHANGED = 0 // send `0` when we don't want to change the weight.
```

Reply:
and we should use these whereever we need to, e.g. `getWeightArgument`

Reply:
Yep done, and used it in the only other spot where it's necessary